Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -742,6 +744,48 @@ public function getPrevMeaningfulToken($index)
     }
 
     /**
+     * Find a sequence of meaningful tokens and returns the array of their locations.
+     *
+     * @param array      $sequence      an array of tokens (same format used by getNextTokenOfKind)
+     * @param int        $startIdx      start index start index
+     * @param int        $endIdx        end index end index
+     * @param bool|array $caseSensitive global case sensitiveness or an array of booleans (for each token)
+     *
+     * @return array|null an array containing the indexes for the sequence elements
+     */
+    public function findSequence(array $sequence, $startIdx, $endIdx, $caseSensitive = true)
```

Comment:
please don't write spaghetti code - whole method is written without any blank line for readability